### PR TITLE
Update docs to reflect change in auth

### PR DESCRIPTION
**Pull request by CelestineKao**
See title.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ Additional features:
...
 The commands in the orb will expose the following environment variables:
 
-* `GITHUB_PR_BASE_BRANCH` - The base branch for the PR.
-* `GITHUB_PR_NUMBER` - The number of the PR.
-* `GITHUB_PR_TITLE` - The title of the PR.
-* `GITHUB_PR_COMMIT_MESSAGE` - The current commit's message.
-* `GITHUB_PR_AUTHOR_USERNAME` - The PR author's username.
-* `GITHUB_PR_AUTHOR_NAME` - The PR author's name.
-* `GITHUB_PR_AUTHOR_EMAIL` - The PR author's email address.
+- `GITHUB_PR_BASE_BRANCH` - The base branch for the PR.
```
Comment (CelestineKao) [inline README.md:15]: huh, looks like i have a plugin that swapped out all the `*` for `-` 

¯\_(ツ)_/¯

**Comment by CelestineKao** [comment]
Build failed because dev version of orb needs to be republished. But I don't think I have permissions to do that. @ns-bdesimone merge merge? And then publish new version to to orb registry? Happy to do it myself if you make me an NS org admin ;)

**Comment by ns-bdesimone** [comment]
> Build failed because dev version of orb needs to be republished. But I don't think I have permissions to do that. @ns-bdesimone merge merge? And then publish new version to to orb registry? Happy to do it myself if you make me an NS org admin ;)

I will take care of the build

**Review by ns-bdesimone — APPROVED** [review]
LGTM

**Comment by CelestineKao** [comment]
thanks fam

**Pull request merged**